log10/cli/cli_commands.py

```
@@ -0,0 +1,71 @@
+try:
+    import click
+except ImportError:
+    print(
```

may want to `raise ImportError` instead of print.